src/content/troubleshooting/_index.en.md

```diff
@@ -217,11 +217,72 @@ Status:
 Events:  <none>
 ```
 
-Note that for a headless Service, the `Ips` field will contain the list of associated Endpoint IPs.
+For headless Service, you need to check `EndpointSlice` resource.
 
 If the data is not correct, you can manually edit the ServiceImport resource to set the correct IP as a workaround and create an
 [issue](https://github.com/submariner-io/lighthouse/issues) with relevant information.
 
 If the ServiceImport `Ips` are correct but still not being returned from DNS queries, check the connectivity to the cluster
 using [`subctl show endpoint`](../deployment/subctl/_index.en.md#show-endpoints). The Lighthouse CoreDNS Server only returns IPs
 from connected clusters.
+
+##### Check EndpointSlice resources
+
+For headless Service, next we check if the EndpointSlice resources were properly created for the service you're
+trying to access. EndpointSlice resources are created in the same namespace as the source Service. The format of a EndpointSlice
+resource's name is as follows:
+
+`<service-name>--<cluster-id>`
+
+Run `kubectl get endpointslices --all-namespaces |grep <your-service-name>` on the Broker cluster to check if a resource was created for
+your service. If not, then check the Lighthouse Agent logs on the cluster where service was created and look for any error or warning
+messages indicating a failure to create the ServiceImport resource for your service. The most common error is `Forbidden` if the RBAC wasn't
+configured correctly. Depending on the deployment method used, 'subctl' or 'helm', it should've been done for you. Create an
+[issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
+
+If the EndpointSlice resource was created correctly on the Broker cluster, the next step is to check if it exists on the cluster where
+you're trying to access the service. Follow the same steps as earlier to get the list of the EndpointSlice resources and check if the
+EndpointSlice for your service exists. If not, check the logs of the Lighthouse Agent on the cluster where you are trying to access the
+service. As described earlier, it will most commonly be an issue with RBAC otherwise create an
+[issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
```

Comment:
```suggestion
If the EndpointSlice resource was created correctly on the Broker cluster, the next step is to check if it exists on the cluster where
you're trying to access the Service. Follow the same steps as earlier to get the list of the EndpointSlice resources and check if the
EndpointSlice for the Service exists. If not, check the logs of the Lighthouse Agent on the cluster where you are trying to access the
Service. As described earlier, it will most commonly be an issue with RBAC so create an
[issue](https://github.com/submariner-io/lighthouse/issues) with relevant log entries.
```